mdx-models/src/main/java/com/mx/path/model/mdx/model/device/VerificationMethod.java

```diff
@@ -0,0 +1,18 @@
+package com.mx.path.model.mdx.model.device;
+
+import lombok.Data;
+
+import com.mx.path.model.mdx.model.MdxBase;
+
+@Data
+public class VerificationMethod extends MdxBase<VerificationMethod> {
+  public enum METHOD {
+    CODE,
+    LINKj
```

Comment:
Is this supposed to be "LINKj" and not "LINK"?

Reply:
Fixed.